src/Cli/dotnet/commands/dotnet-list/dotnet-list-package/ListPackageReferencesCommandParser.cs

```diff
@@ -61,6 +61,11 @@ internal static class ListPackageReferencesCommandParser
                 ArgumentHelpName = CommonLocalizableStrings.LevelArgumentName
             }.ForwardAsSingle(o => $"--verbosity:{o}");
 
+        public static readonly Option FormatOption = new ForwardedOption<string>("--format", LocalizableStrings.CmdFormatDescription)
```

Comment:
I think you might want to use an enum or something here so that the parser knows what the valid values are.